pegasus-spark-analyser/src/main/scala/com/xiaomi/infra/pegasus/spark/analyser/PegasusWriter.scala

```diff
@@ -0,0 +1,141 @@
+package com.xiaomi.infra.pegasus.spark.analyser
+
+import java.time.Duration
+
+import com.xiaomi.infra.pegasus.tools.{FlowController, Tools}
+import org.apache.spark.TaskContext
+import org.apache.spark.rdd.RDD
+import org.slf4j.LoggerFactory
+
+import scala.collection.JavaConverters._
+import com.xiaomi.infra.pegasus.client.{
+  ClientOptions,
+  PException,
+  PegasusClientFactory,
+  SetItem
+}
+
+case class WriteConfig() extends Serializable {
+  var metaServer: String = _
+  var timeout: Long = 1000
+  var asyncWorks: Int = 4
+
+  var cluster: String = _
+  var table: String = _
+
+  // ttlThreshold(unit is seconds) must not be negative. less than the threshold won't be write,
+  // default is 0.
+  var ttlThreshold: Int = 0
+  // flow control：
+  // single_qps = flowControl * bulkNum
+  // total_qps = single_qps * partitionNum(here partitionNum equal count of partition in
+  // running, also is count of parallel jobs, which isn't more than total pegasus partition count)
+  var bulkNum: Int = 10
+  var flowControl: Int = 5
+
+  def setMetaServer(metaServer: String): WriteConfig = {
```

Comment:
scala auto-generates setter and getter by default for `var` type.

Reply:
`auto-generates` should not `return this`?